www/src/components/headCommon.astro

```diff
@@ -14,25 +14,3 @@
     element.setAttribute("tabindex", "0");
   });
 </script>
-
```

Comment:
when the themetoggler is no longer react we can move this part to the toggler itself which makes more sense imo